packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import IsDarkContext from "../../context";
+import { MAXIMUM_NUM_OF_CHARACTER } from "../../constants";
+
+import * as sc from "./Link.styles";
+
+const truncate = str =>
+  str.length > MAXIMUM_NUM_OF_CHARACTER ? `${str.substring(0, MAXIMUM_NUM_OF_CHARACTER)}...` : str;
+
+function isString(item) {
+  return typeof item === "string" || item instanceof String;
+}
+
+function Link(props) {
+  const { children, href, as, ...moreProps } = props;
+  const shouldTruncate = isString(children) && children.length > MAXIMUM_NUM_OF_CHARACTER;
+
+  const link = (
+    <IsDarkContext.Consumer>
+      {isDark => (
+        <sc.Link data-pka-anchor="breadcrumbs.link" as={as} kind="minor" href={href} isDark={isDark} {...moreProps}>
+          {shouldTruncate ? truncate(children) : children}
+        </sc.Link>
+      )}
+    </IsDarkContext.Consumer>
+  );
+
+  return (
+    <sc.ListItem data-pka-anchor="breadcrumbs.listItem">
+      {shouldTruncate ? (
```

Comment:
When I sad this in storybook before reading the code, I was like how she is doing it so can detect if the texts have ellipsis or not with css :D. 

Now I can see the trick 😄